### DISCUSSION TITLE
Multicore Support

### DISCUSSION THREAD

**Discussion by gmfricke** [GeoStat-Framework/PyKrige#193] [Q&A]
Hello,

Does PyKrige support parallelization? Presumably is gets some for free by using openmp though numpy, but are there any pykrige specific options? 

All the best,

Matthew

**Comment by MuellerSeb** [comment]
Hey there,

Since PyKrige will reuse the implementations of GSTools in the future (>v2), parallel support will come with it ([link](https://geostat-framework.readthedocs.io/projects/gstools/en/v1.3.1/#pip))
I will keep an eye on parallel support (especially using `prange` in cython loops, as done in all GSTools cython modules) for future implementations.

Do you have any further suggestions on how to use parallelization techniques?

Cheers,
Sebastian
